frontend/src/components/PaginationElement.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import classNames from 'classnames'
+
+const PaginationElement = ({ page, isActive, changePage }) => {
+  const className = classNames({
+    pagination__page: true,
+    '-active': isActive,
+    '-dot': page === '...',
+  })
```

Comment:
If `pagination__page` is supposed to always present, what do you think about doing it this way:
```suggestion
  const className = classNames("pagination__page", 
  { 
    '-active': isActive,
    '-dot': page === '...',
  })
```